File: sample-v7/src/main/java/io/erva/sample/home/MenuItemCell.java
package io.erva.sample.home;

import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import io.erva.celladapter.Layout;
import io.erva.celladapter.v7.Cell;
import io.erva.sample.BaseCell;

@Layout(android.R.layout.simple_list_item_1)
public class MenuItemCell extends BaseCell<MenuItemModel, Cell.Listener<MenuItemModel>> {

    @BindView(android.R.id.text1) TextView titleTv;

    public MenuItemCell(View view) {
        super(view);
    }

    @Override
    protected void bindView() {
        titleTv.setText(getItem().titleId);
    }
}

File: sample-x-kotlin/src/main/kotlin/io/erva/sample/home/MenuItemCell.kt
package io.erva.sample.home

import android.view.View
import android.widget.TextView
import io.erva.celladapter.Layout
import io.erva.celladapter.x.Cell

@Layout(android.R.layout.simple_list_item_1)
class MenuItemCell(view: View) : Cell<MenuItemModel, Cell.Listener<MenuItemModel>>(view) {

    override fun bindView() {
        val item = item()
        (view.findViewById(android.R.id.text1) as TextView).setText(item.titleId)
    }
}